changelog/fragments/liveness_readiness_probe_for_operator.yaml

```
@@ -0,0 +1,67 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Ansible/Helm-based operators, add Liveness and Readiness probe by default using [sigs.k8s.io/controller-runtime/pkg/healthz#CheckHandler](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/healthz#CheckHandler).
```

`CheckHandler` is just an interface. The simple implementation actually used is `healthz.Ping`. Also for brevity just include the package name not full path when a link is included.
```suggestion
      For Ansible/Helm-based operators, added Liveness and Readiness probe by default using [`healthz.Ping`](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/healthz#CheckHandler).
```